### PR TITLE
Cluster-autoscaler: check node/mig state

### PR DESCRIPTION
**Pull request by mwielgus**
cc: @piosz @fgrzadkowski

### CODE REVIEW

File: cluster-autoscaler/utils.go [hunk 1]
```diff
@@ -105,3 +109,35 @@ func GetOldestFailedSchedulingTrail(pods []*kube_api.Pod) *time.Time {
...
+		if err != nil {
+			return err
+		}
+
+		migConfig, err := gceManager.GetMigForInstance(instanceConfig)
+		if err != nil {
+			return err
+		}
+		url := migConfig.Url()
+		count, _ := migCount[url]
```
Comment (piosz) [inline cluster-autoscaler/utils.go]: nit: you don't need the second variable here

**Comment by piosz** [comment]
LGTM, just a nit

**Comment by piosz** [comment]
:facepunch: :facepunch: :facepunch: 
```
Verifying hack/../hack/verify-golint.sh
FAILED
```

File: cluster-autoscaler/utils.go (hunk 1 above)
Comment (mwielgus) [inline cluster-autoscaler/utils.go]: Done.

**Pull request merged**